src/screens/FeedScreen/Content.js

```diff
@@ -264,7 +268,10 @@ const Content = ({
   };
 
   return (
-    <Pressable onPress={isBlurredPost ? null : () => onPress()} style={[styles.contentFeed, style]}>
+    <Pressable
+      onLayout={handleHeightContainer}
+      onPress={isBlurredPost ? null : () => onPress()}
+      style={[styles.contentFeed, style]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onLayout` prop has been added to the `Pressable` component to call the `handleHeightContainer` function when the layout changes. This is a good practice as it allows the component to adapt to different content sizes dynamically.
```diff
- <Pressable onPress={isBlurredPost ? null : () => onPress()} style={[styles.contentFeed, style]}>
+ <Pressable
+   onLayout={handleHeightContainer}
+   onPress={isBlurredPost ? null : () => onPress()}
+   style={[styles.contentFeed, style]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->